.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: groovy
```

Comment:
Sure, we can change it anytime. The `language` parameter doesn't matter, but I needed to put something as it is required by Travis.
Thanks for the upvote.